image_generation/stable_diffusion_1_5/cpp/README.md

```diff
@@ -104,13 +116,13 @@ Positive prompt: cyberpunk cityscape like Tokyo New York  with tall buildings at
 
 Negative prompt: (empty, check the [Notes](#negative-prompt) for details)
 
-Read the numpy latent instead of C++ std lib for the alignment with Python pipeline
+To read the numpy latent instead of C++ std lib for the alignment with Python pipeline, use `-r, --readNPLatent` argument.
 
 * Generate image without lora `./build/stable_diffusion -r`
 
    ![](./without_lora.bmp)
 
-* Generate image with soulcard lora `./build/stable_diffusion -r`
+* Generate image with soulcard lora `./build/stable_diffusion -r -l path/to/soulcard.safetensors`
 
    ![](./soulcard_lora.bmp)
 
```

Comment:
While you are nearby, please replace `is differ` to `differ` below.

Reply:
Fixed for both SD and LCM